store/storetest/team_store.go

```diff
@@ -1082,25 +981,19 @@ func testSaveTeamMemberMaxMembers(t *testing.T, ss store.Store) {
 		panic(teamErr)
```

Comment:
You missed this one :wink: Fixing this will also fix the build error

Reply:
fixed

Reply:
Would you please remove the `panic` call and replace it with a `require.Nil` call?

Reply:
done